glusterd2/xlator/options/options.go

```diff
@@ -186,12 +187,29 @@ func ValidateInt(o *Option, val string) error {
 
 // ValidateInternetAddress validates the Internet Address
 func ValidateInternetAddress(o *Option, val string) error {
-	return ErrInvalidArg
+	_, _, err := net.ParseCIDR(val)
+	if err != nil {
+		return ErrInvalidArg
+	} else if validate.IsIP(val) != true {
```

Comment:
Is `validate.IsIP` taking consideration of validating both Ipv4 and IPv6?

Reply:
@atinmu yes, validate.IsIP validates IPv4 and IPv6 addresses too.